addons/l10n_in/models/account_invoice.py

```diff
@@ -22,7 +22,7 @@ def _compute_amount_total_words(self):
             ('overseas', 'Overseas'),
             ('special_economic_zone', 'Special Economic Zone'),
             ('deemed_export', 'Deemed Export')
-        ], string="GST Treatment", readonly=True, states={'draft': [('readonly', False)]})
+        ], string="GST Treatment", readonly=False, states={'draft': [('readonly', False)]}, compute='_compute_l10n_in_gst_treatment')
```

Comment:
This is a must compute with the store because the user can also able to change this value and in v14 it's already stored field so you can't change the property of a field in the stable version.